file-formats/sajt/README.md

```diff
@@ -1,5 +1,5 @@
 # SAJT File Format
 
 File | Cardinality | Definition | Schema | Example
-:--- | :---  | :--- | :--- | :---
-`<name>.sajt.json` | 1 | [`zif_aff_sajt_v1.intf.abap`](./type/zif_aff_sajt_v1.intf.abap) | [`sajt-v1.json`](./sajt-v1.json) | [`z_aff_example_sajt.sajt.json`](./examples/z_aff_example_sajt.sajt.json)
+:--- | :--- | :--- | :--- | :---
+`<name>.sajt.json` | 1 |  [`zif_aff_sajt_v1.intf.abap`](./type/zif_aff_sajt_v1.intf.abap)  | [`sajt-v1.json`](./sajt-v1.json) | [z_aff_example_sajt.sajt.json](./examples/z_aff_example_sajt.sajt.json)
```

Comment:
Did you generate the output of the readme file using our program? 
If so, there seems to be an issue with the generator for the readme file.

```suggestion
`<name>.sajt.json` | 1 | [`zif_aff_sajt_v1.intf.abap`](./type/zif_aff_sajt_v1.intf.abap) | [`sajt-v1.json`](./sajt-v1.json) | [`z_aff_example_sajt.sajt.json](./examples/z_aff_example_sajt.sajt.json)
```

Reply:
Yes, the file was generated by report SAFF_GENERATE_REPO

Reply:
I suggest to fix it manually in this PR by committing my suggestion above. We need to fix the generator.

Reply:
opened issue https://github.com/SAP/abap-file-formats-tools/issues/299. Still need to be fixed manually here.

Reply:
I have created also an issue to fix the generator SAP/abap-file-formats-tools#300

Reply:
I changed the file